resources/sdk/purecloudgo/templates/apiclient.mustache

```diff
@@ -118,8 +118,15 @@ func (c *APIClient) CallAPI(path string, method string,
 	// Set post body
 	if postBody != nil {
 		request.Header.Set("Content-Type", "application/json")
-		j, _ := json.Marshal(postBody)
-		request.SetBody(ioutil.NopCloser(bytes.NewReader(j)))
+		if c.configuration.PatchConfiguration != nil && method == "PATCH" && c.configuration.PatchConfiguration.RemoveOmitEmpty && pathContainsAPI(path, c.configuration.PatchConfiguration.APIs) {
+			processedModel, _ := processModel(postBody)
```

Comment:
A couple of small things here:

1.  You have a couple of fmt.Println.  These should be logging debug statements.
2. Also you might want to explicitly call out that patch overrides are present before you process the model.  Otherwise, if a customer or support is debugging why their fields are overridden they can explicitly see if it is because the patch override models fields have been defined
3. Also you are ignoring the error here.  You should handle this gracefully and log the error and breakout out of the code.  With the way you have written this now the processedModel value will be NULL and you will be setting the SetBody with a nil value, which may cause a panic later on down the line.  

In go errors should be explicitly handled and ignoring an error value should be the exception instead of the norm.